### PR TITLE
6927: updated button for room selector

### PR DESCRIPTION
**Pull request by nikola-gladovic**
Signed-off-by: Nikola <nikola.gladovic@nextcloud.com>

#6927

### CODE REVIEW

**Review by marcoambrosini — DISMISSED** [review]
tested and works

**Comment by nickvergessen** [comment]
Test is failing:
```
FAIL src/views/RoomSelector.spec.js
  ● RoomSelector.vue › emits select event on select

    [vue-test-utils]: find did not return button, cannot call trigger() on empty Wrapper

      141 | 		await list.at(1).trigger('click')
      142 | 
    > 143 | 		await wrapper.find('button').trigger('click')
          | 		                             ^
      144 | 
      145 | 		expect(eventHandler).toHaveBeenCalledWith('token-3')
      146 | 	})

      at throwError (node_modules/@vue/test-utils/dist/vue-test-utils.js:1700:9)
      at ErrorWrapper.trigger (node_modules/@vue/test-utils/dist/vue-test-utils.js:9000:3)
      at _callee3$ (src/views/RoomSelector.spec.js:143:32)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)
      at Generator.invoke [as _invoke] (node_modules/regenerator-runtime/runtime.js:294:22)
      at Generator.next (node_modules/regenerator-runtime/runtime.js:119:21)
      at asyncGeneratorStep (src/views/RoomSelector.spec.js:15:103)
      at _next (src/views/RoomSelector.spec.js:17:194)
```

You can run them manually with:
```
npm run test:unit
```

I think in the failing test you need to replace `.find('button')` with something like `.findComponent( name: 'Button' )`

**Review by nickvergessen — APPROVED** [review]

File: src/views/RoomSelector.vue [hunk 1]
```diff
@@ -165,9 +167,10 @@ export default {
 
 <style lang="scss" scoped>
 #modal-inner {
-	width: 90vw;
-	max-width: 400px;
-	height: 55vh;
+	width: 100%;
+	height: 100%;
+	padding: 20px;
+	margin: 0 auto;
```
Comment (nickvergessen) [inline src/views/RoomSelector.vue:173]: Good @marcoambrosini ?

Comment (nikola-gladovic) [reply src/views/RoomSelector.vue:173]: Hi @marcoambrosini 
Would you like to check it? :)

**Comment by nikola-gladovic** [comment]
image

File: src/views/RoomSelector.vue [hunk 2]
```diff
@@ -165,9 +167,10 @@ export default {
 
 <style lang="scss" scoped>
 #modal-inner {
-	width: 90vw;
-	max-width: 400px;
-	height: 55vh;
+	width: 100%;
+	height: 100%;
+	padding: 20px;
```
Comment (marcoambrosini) [inline src/views/RoomSelector.vue:172]: After seing the screenshot I think we should dial this down to 12px

**Review by marcoambrosini — CHANGES_REQUESTED** [review]

File: src/views/RoomSelector.vue (hunk 2 above)
Comment (nikola-gladovic) [reply src/views/RoomSelector.vue:172]: @marcoambrosini 
It should be:
        height: 55vh;
	width: 100%;
	height: 100%;
	padding: 12px;
	
Correct? :)

Comment (marcoambrosini) [reply src/views/RoomSelector.vue:172]: Correct!

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**